dialogues/risawoz/db/zh/attraction_db.json

```diff
@@ -0,0 +1,562 @@
+[
+  {
```

Comment:
We need the database for English too

Reply:
I just sent you an English version of database via email. However, there are some untranslated non-slot attributes and values, which may need some time to translate manually. You can have a try to see if the current version is good for training.